app/Http/Controllers/api/CollaboratorsController.php

```diff
@@ -60,7 +60,6 @@ public function uniqueCollaborators(...$arrays)
 
     public function collaboratorLanding()
```

Comment:
We need to change the method name if suggestion at `index()` is accepted.

```suggestion
    public function collaboratorItaLanding()
```